##############################################################################
# Worlds
##############################################################################

worlds:
  robotworld15:
    display_name: 'Robotworld 2015'
    description: "Forth and back in Yujin Robot's booth."
    default: true

##############################################################################
# Locations
##############################################################################

semantic_locations:
  homebase:
    name: 'Homebase'
    description: 'At the bar'
    world: robotworld15
    pose: { x: 0.0, y: 0.00, theta: 0.00 }
    keyframe_id: 0
  table_close:
    name: 'Table close'
    description: 'The table close to the homebase (in front of the door)'
    world: robotworld15
    pose: { x: -0.15, y: -3.00, theta: -1.57 }
    keyframe_id: 1
  table_far:
    name: 'Table far'
    description: 'The table far away from the homebase (behind the door)'
    world: robotworld15
    pose: { x: -0.21, y: -5.20, theta: -1.57 }
    keyframe_id: 2

# For testing purposes only
docking_stations:
  'home_dock':
    primary_id: 9
    left_id: 10
    right_id: 11
    pose:
      x : 9.0
      y : -1.0
      theta : -1.57
